skyrimnet_chatterbox/chatterbox/models/voice_encoder/voice_encoder.py

```
@@ -1,361 +1,361 @@
+        # Pass the input through the LSTM layers
+        _, (hidden, _) = self.lstm(mels)
+
+        # Project the final hidden state
+        raw_embeds = self.proj(hidden[-1])
+        if self.hp.ve_final_relu:
+            raw_embeds = F.relu(raw_embeds)
+
+        # L2 normalize the embeddings.
+        return raw_embeds / torch.linalg.norm(raw_embeds, dim=1, keepdim=True)
+
+    def inference(self, mels: torch.Tensor, mel_lens, overlap=0.5, rate: float=None, min_coverage=0.8, batch_size=None):
```

Ruff (0.14.10)

PEP 484 prohibits implicit `Optional`

Convert to `T | None`

(RUF013)